scio-avro/src/main/scala/com/spotify/scio/avro/schemas/package.scala

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2023 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.scio.avro
+
+package object schemas extends instances.AvroInstances
```

Comment:
How this impacts the typical usage?

Reply:
I don't think there's really any users of this schemas package to speak of.

But the change would be if you are using something from this package, you'd need to add:
```
import com.spotify.scio.avro.schemas._
```
In addition to the existing:
```
import com.spotify.scio.schemas._
```